### PR TITLE
Cultist Service Borgs

### PR DESCRIPTION
**Pull request by ghost**
I feel like I've made like 3 PR's for this, so feel free to read old PR's. It's fully functional.

:cl:
 * rscadd: Re-added cultist borgs, now purchasable at the black market.

### CODE REVIEW

File: code/datums/gamemode/factions/bloodcult/Friendly Cult/friendlycult_runespells.dm [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+		else
+			invoke(activator,invocation)
+			cast()
+
+	
+	
+	
+/datum/rune_spell/friendly_cult/conjurecookie
+	name = "Conjure Cookie"
+	desc = "This simple rune summons a bowl of cookies filled with O+, Nar-Sie approved cookies."
```
Comment (bathosbathtime) [inline code/datums/gamemode/factions/bloodcult/Friendly Cult/friendlycult_runespells.dm:29]: The cookies are o-, as they should be, not o+

**Comment by boy2mantwicethefam** [comment]
MAKE IT CULTIST TG BORG

**Comment by tuypo1** [comment]
I loved the cultist borgs but I cannot abide them being on the black market or being a service borg thing instead of a hug borg thing

**Comment by ghost** [comment]
The problem with items being RNG pickup (hugborg) is that players who actually want to use an item (AKA the good rpers who will do interesting stuff with it) will rarely get it, while retards who barely managed to boot up this malware client will waste the "round where a hugborg is found and some guy was bothered enough to get and install the cult upgrade roll". 
I don't mind changing the way you get it to require more effort but hugborg exclusivity would be a mistake, I feel.

**Comment by tuypo1** [comment]
That's a fair argument I suppose
And honestly my problem with it being on the black market is that I hate the black market in general, if we have to have a black market though it's honestly probably the best place for this sort of thing

**Comment by ghost** [comment]
Yeah well we can always change it later. What if I were to make it an illegal research upgrade that costs phazon, similar to the honk upgrade that costs bananium

File: code/datums/gamemode/factions/bloodcult/Friendly Cult/friendlycult_runespells.dm (hunk 1 above)
Comment (ghost) [reply code/datums/gamemode/factions/bloodcult/Friendly Cult/friendlycult_runespells.dm:29]: O-K

**Comment by tuypo1** [comment]
Could be cool

**Comment by Blithering** [comment]
I think there's a precedent for non-hugborg modules getting these kind of upgrades, the warden and detective upgrades can be put on secborgs if I remember right, and they get the hugborg icon when changed. I don't see a problem with it working with hugborgs and service borgs both.

**Comment by Blithering** [comment]
cultborg
I did a thing so we can tell which borgs are spooky.

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 2]
```diff
@@ -321,7 +321,25 @@
...
+	desc = "Used to allow a service cyborg to assist in peaceful following of occult religion. The silicon \"conversion\" ritual affects the controlling mind directly and is non-reversible."
+	icon_state = "cyborg_upgrade2"
+	required_modules = list(SERVICE_MODULE)
+	modules_to_add = list(/obj/item/weapon/reagent_containers/food/drinks/cult/cyborg, /obj/item/weapon/reagent_containers/syringe/robot/cultist)
 
+/obj/item/borg/upgrade/cult/attempt_action(var/mob/living/silicon/robot/R,var/mob/living/user)
+	if(..())
+		return FAILED_TO_ADD
+	
+	R.cult_permitted = 1
```
Comment (jknpj) [inline code/game/objects/items/robot/robot_upgrades.dm:336]: We already talked about consistency with binary checks in cyborg code.

**Review by jknpj — COMMENTED** [review]
Grab those icons with blithering and make the upgrade a hugborg+service upgrade like the Noir/Watch/HoS ones.

**Comment by ghost** [comment]
@Blithering Do you have the dmi or all 4 directional sprites for that? Does it have a version for service borg?

**Comment by Blithering** [comment]
I have all the directional sprites yes, I don't have a version of it for service because there's so many service borg types I'd be at this weeks, hence why all the gimmick modules use hugborg's sprite as their base (warden, detective, clown).

**Comment by ghost** [comment]
I'll have service borgs use the hugborg model then.

**Comment by SonixApache** [comment]
No, dummy, you'll have the cultborgs be hugborgs.

**Pull request closed without merging**